pom.xml

```diff
@@ -23,13 +23,12 @@
 
     <parent>
         <groupId>org.apache.sling</groupId>
-        <artifactId>sling</artifactId>
-        <version>34</version>
+        <artifactId>sling-bundle-parent</artifactId>
+        <version>47</version>
```

Comment:
in my opinion, we can directly goto version `48`

Reply:
@ashokmca07 Yes, that is probably true, but I would prefer to deal with that later since it is not critical to this PR.  I always try to update to the latest parent version before producing a release candidate but we are not there yet.